soweego/commons/constants.py

```diff
@@ -164,15 +162,22 @@
 WIKIDATA_API_SESSION = 'wiki_api_session.pkl'
 SHARED_FOLDER = '/app/shared/'
 
+
 # Supervised classification
+SVC_CLASSIFIER = 'support_vector_machines'
+NAIVE_BAYES_CLASSFIER = 'naive_bayes'
```

Comment:
```suggestion
NAIVE_BAYES_CLASSIFIER = 'naive_bayes'
```
Typo detector enabled 😄